docs/pages/reference/config.mdx

```diff
@@ -523,6 +523,12 @@ auth_service:
       # 2 or more.
       agent_connection_count: 1
 
+    # Tells tsh to load the host CAs of all clusters when trying to ssh into a Teleport Node,
+    # instead of just the host CA for the current cluster. This may be useful for
+    # users that want to log in to a root cluster and then "tsh ssh" into a node
+    # in a leaf cluster. Defaults to false.
+    load_all_host_cas: false
```

Comment:
Let's remove "host" from here also like my previous comment. Also, everywhere in the code too, instead of `LoadAllHostCAs` let's just rename to `LoadAllCAs`.
```suggestion
    load_all_cas: false
```